feature/idCheck

nahokyun opened the pull request:
입력한 id가 사용이 가능한 아이디인지 확인하는 메소드인 isId_Duplicated를 추가하였고 만일 중복된 아이디가 존재한다면 409 Conflict를 응답하고 중복되지않은 아이디라면 200을 반환하게 구현하였습니다.

nahokyun committed:
feature/idCheck
입력한 id가 사용이 가능한 아이디인지 확인하는 메소드인 isId_Duplicated를 추가하였고 만일 중복된 아이디가 존재한다면 409 Conflict를 응답하고 중복되지않은 아이디라면 200을 반환하게 구현하였습니다.

sunwootest commented on src/main/java/study/movieservice/controller/MemberController.java:
```diff
@@ -18,4 +18,11 @@ public ResponseEntity<Void> join(@RequestBody Member member){
     }
+
+    @PostMapping("/id-check")
+    public ResponseEntity<String> isId_Duplicated(@RequestParam String loginId){
```
CamelCase 지켜주세요

nahokyun replied:
isIdDuplicated로 메소드명 변경하였습니다.

sunwootest commented on src/main/java/study/movieservice/service/MemberService.java:
```diff
@@ -20,4 +20,10 @@ public void save(Member member){
         memberMapper.save(member1);
     }
+
+    public boolean findByLoginId(String loginId){
```
loginId에 null 값이 입력되면 어떻게 처리하죠?

nahokyun replied:
추후 입력값들에 대한 유효성검사를 이슈로 묶어서 처리하겠습니다.

nahokyun committed:
feature/idCheck_na

mapper의 findByloginId와 controller의 isId_Duplicated를 camel case에 맞춰서 변경하였습니다.
idIdDuplicated의 파라미터를 Json형식으로 받는것으로 변경하였습니다.

nahokyun committed:
Merge branch 'feature/signup_na' into feature/idCheck_na

sunwootest commented:
임시 approve 드립니다. PR 코멘트를 달아주지 않으면 코드리뷰를 반영했는지 확인하기 어렵습니다 :)
